jenkins/e2e-image/e2e-runner.sh

```diff
@@ -243,8 +243,10 @@ function setup_gci_vars() {
 
 ### Pre Set Up ###
 if running_in_docker; then
-    record_command "${STAGE_PRE}" "download_gcloud" curl -fsSL --retry 3 --keepalive-time 2 -o "${WORKSPACE}/google-cloud-sdk.tar.gz" 'https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz'
-    install_google_cloud_sdk_tarball "${WORKSPACE}/google-cloud-sdk.tar.gz" /
+    if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
```

Comment:
Also, `bootstrap.py` [sets up `GOOGLE_APPLICATION_CREDENTIALS`](https://github.com/kubernetes/test-infra/blob/746ff9bc58621a7365a0842c059900a7a0d7e783/jenkins/bootstrap.py#L544), so anything run through bootstrap right now will stay the same.